test/sequential/feeLock.API.feePayment.test.ts

```diff
@@ -0,0 +1,399 @@
+/*
+ *
+ * @group sequential
+ * @group gassless
+ */
+
+import { Keyring } from "@polkadot/api";
+import { getApi, initApi, getMangataInstance } from "../../utils/api";
+import { Assets } from "../../utils/Assets";
+import { MGA_ASSET_ID } from "../../utils/Constants";
+import { waitSudoOperataionSuccess } from "../../utils/eventListeners";
+import { BN, toBN } from "@mangata-finance/sdk";
+import { setupApi, setupUsers } from "../../utils/setup";
+import { Sudo } from "../../utils/sudo";
+import { updateFeeLockMetadata, unlockFee } from "../../utils/tx";
+import { AssetWallet, User } from "../../utils/User";
+import { getEnvironmentRequiredVars, waitForNBlocks } from "../../utils/utils";
+import { Xyk } from "../../utils/xyk";
+
+jest.spyOn(console, "log").mockImplementation(jest.fn());
+jest.setTimeout(2500000);
+process.env.NODE_ENV = "test";
+
+const { sudo: sudoUserName } = getEnvironmentRequiredVars();
+let testUser1: User;
+let sudo: User;
+let keyring: Keyring;
+let firstCurrency: BN;
+let feeLockMetadata: any;
+let periodLength: any;
+let feeLockAmount: any;
+let whitelistedTokens: any[];
+const thresholdValue = new BN(30000);
+const defaultCurrencyValue = new BN(10000000);
+const defaultPoolVolumeValue = new BN(1000000);
+
+async function checkErrorSellAsset(
```

Comment:
Plz split into two functions

Reply:
I stay only the first part because the second part uses only one test.